Resources/public/webcomponents/ez-universaldiscovery.html

```diff
@@ -0,0 +1,39 @@
+<link rel="import" href="../../../assets/ezplatform/polymer/polymer-element.html">
+<link rel="import" href="../../../webcomponents/ez-yui-app.html">
+
+<style>
+    ez-universal-discovery {
+        display: block;
+        background: rgba(0, 0, 0, 0.4);
+
+        /* taken from universaldiscovery.css .ez-universaldiscovery-container */
+        position: fixed;
+        width: 100%;
+        height: 100%;
+        box-sizing: border-box;
+        padding: 2% 3%;
+        top: 0;
+        left: 0;
+    }
+
+    ez-universal-discovery .ez-view-universaldiscoveryview {
+        background: #fff;
+        width: 100%;
+        height: 100%;
+        overflow: auto;
+    }
+
+    ez-universal-discovery a {
+        cursor: pointer;
+    }
+
+    /* missing from modules/tabs.css (and coming for theme/modules/tabs.css) */
```

Comment:
but where are the CSS included in https://github.com/ezsystems/hybrid-platform-ui/blob/master/src/bundle/Resources/public/webcomponents/ez-universaldiscoverywidget.html#L31 ?